Jenkinsfile.integration

```diff
@@ -56,12 +57,17 @@ pipeline {
                     sh "git fetch --no-tags"
                     List<String> sourceChanged = sh(returnStdout: true, script: "git diff --name-only origin/master..origin/${env.BRANCH_NAME}").split()
                     echo "Files changed for this PR:\n${sourceChanged.join('\n')}"
+                    FullDeploymentNeeded = false
                     for (int i = 0; i < sourceChanged.size(); i++) {
                         if (sourceChanged[i].contains("playbooks/")) {
                             PlaybooksChanged = true
                         }
                         if (sourceChanged[i].contains("doc/")) {
                             DocsChanged = true
+                        } else {
```

Comment:
is the else only triggering when doesn't contain docs? (#me, noob at jenkins...).
If that's the case, what about a change that contain both docs and code? Wouldn't that be a problem?

Reply:
`sourceChanged` is a list of strings. It contains all name of files that have been touched by the current PR (generated by the git diff command on line 58).

As we are inside a for loop (line 61) iterating through that list. The else branch will trigger as soon as there is a single filename in that list that does not contain `doc/`. (So if there is a change that touches doc and code it would trigger)

Reply:
Basic logic doesn't apply on Friday afternoon, everyone knows that.

I suppose the reno would be considered as part of docs so I don't see a reason to not do it this way.